vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -1043,50 +1042,56 @@ public void updateTabStyle(final int primaryColor,
                                final int fabColor,
                                final int fabPressedColor) {
         // Apply primary color
-        mToolbar.setBackgroundColor(primaryColor);
+        mToolbar.setBackgroundResource(R.drawable.act_gradient);
+        //mToolbar.setBackgroundColor(primaryColor);
         mVectorPendingCallView.updateBackgroundColor(primaryColor);
-        mSyncInProgressView.setBackgroundColor(primaryColor);
+        if(!getResources().getBoolean(R.bool.use_progressbar_original_background))
+            mSyncInProgressView.setBackgroundColor(primaryColor);
```

Comment:
Please surround the then block in parentheses

Reply:
done